src/plugins/csvstorage/csvstorage.c

```diff
@@ -122,11 +128,10 @@ static int csvRead(KeySet *returned, Key *parentKey, char delim, short useHeader
 	}
 	if(!fgets(lineBuffer, length, fp))
 	{
-		// TODO: add line information
```

Comment:
there is no real information on why it's failed available at this point because it's the first read access to the file.
I'd assume that mentioning line 0 at the error would just cause confusion.